### ISSUE TITLE
Declare New Keywords

### ISSUE THREAD

**Issue by zcysxy** [ytliu74/obsidian-pseudocode#9]
Thanks for this plugin! Is it possible to rename a keyword or even define a new keyword? For example, using
`\renewcommand{\algorithmicinput}{\textbf{initialize}}`. Maybe this is a limitation of pseudocode.js.

**Comment by ytliu74** [comment]
Yeah, you are right. This is indeed a limitation of `pseudocode.js`. The customizable macros or preamble file provided by this plugin only works for math blocks (enclosed as `$...$`) inside pseudocode block.

But the `pseudocode.js` used in the plugin has already been modified for some feature by me, so I'll try to find out way to help support keyword renaming. But you can just refer to other supported keywords now as a workaround.

**Comment by ytliu74** [comment]
I just re-checked `pseudocode.js`. Here's an official test case:
![image](https://github.com/ytliu74/obsidian-pseudocode/assets/58876108/6202a1af-7acc-4833-9286-d2db6ca9e289)
Commands like `\textbf{}` is supported directly by `pseudocode.js`, so you can just use `\textbf{initialize}` as a workaround.
I do try to implement this `\renewcommand{\algorithmicxxx}{}` feature, only to find it's quite ugly to implement on the framework of `pseudocode.js`. So I am not planning to implement this, sorry about that. Please refer to upper suggestion instead.

**Comment by zcysxy** [comment]
Sure! Thanks for the suggestion!
